jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/DekorateUtilTest.java

```diff
@@ -15,36 +15,37 @@
 
 import org.eclipse.jkube.kit.common.Dependency;
 import org.eclipse.jkube.kit.common.JavaProject;
-import org.junit.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
 
-import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import java.util.stream.Stream;
 
-public class DekorateUtilTest {
-  @Test
-  public void useDekorate_withNoDekorateDependency_returnsFalse() {
-    // Given
-    JavaProject javaProject = JavaProject.builder().build();
+import static org.assertj.core.api.Assertions.assertThat;
 
-    // When
-    boolean result = DekorateUtil.useDekorate(javaProject);
+class DekorateUtilTest {
 
-    // Then
-    assertThat(result).isFalse();
-  }
-
-  @Test
-  public void useDekorate_withDekorateDependency_returnsTrue() {
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("data")
+  void useDekorateTest(String testDesc, Dependency dependencies, boolean expected) {
     // Given
-    JavaProject javaProject = JavaProject.builder()
-      .dependency(Dependency.builder()
-        .groupId("io.dekorate").artifactId("kubernetes-annotations").version("0.10.5")
-        .build())
-      .build();
-
+    JavaProject javaProject = null;
+    if (dependencies != null) {
```

Comment:
I think if you use `List<Dependency>` as parameter you can get rid of this `if { ... }` block. You can pass emptylist when there is no dependency and singleton list when there is one.